### ISSUE TITLE
Add '/fl' as an alias for '/friends list' in lobby

### ISSUE THREAD

**Issue by MiginsMC** [Minehut/Meta#193]
### Feature request

**Checklist**
- [x] I have read and agree to the README (found on https://github.com/Minehut/Meta)
- [x] This is not a bug report, support request or plugin request
- [x] I have searched the Issue Tracker to make sure this is not a duplicate issue
- [x] This issue does not contain any sensitive information 

**Platform**
Java

**Describe the feature**
Have the command `/fl` be an alias for `/friends list`.

**How the feature is useful**
I personally use `/friends list` a lot and it would benefit me (and probably quite a few others) if this command was shorter.

---
**Use 👍 and 👎 reactions to indicate you agree or disagree with this suggestion! Feel free to discuss the idea in the comments.**

**Comment by MiginsMC** [comment]
@recalibrated and @MisterBasic
I'm interested in why you downvoted my suggestion.

**Comment by theminecoder** [comment]
We generally refrain from adding aliases to commands in the proxy as we don't want to clash with commands on the player server where possible (this is why /f also doesn't work for the global friends command).

**Comment by MiginsMC** [comment]
I realise you've closed this, but I wouldn't be against the command simply being in the lobby, as I use it a lot there, too :)

**Comment by wooden-utensil** [comment]
@MiginsMC a /l command leading to /lobby would be helpful but as minecoder said it might clash with commands on the player server, maybe have a toggle for that?

**Comment by jellz** [comment]
Migins is asking for the alias to only be in the lobby, where commands can't clash with player server commands

**Comment by wooden-utensil** [comment]
oh yea i didn't read =P

**Comment by LukasmanMHdude** [comment]
I mean, with tab complete you can type in /fl and hit tab, I don't feel that its worth it to add a lobby only command for one or 2 more keys pressed.

**Comment by Sniped** [comment]
Yeah, I agree with Lukas. I don’t see the need in implementing it when it’s just two arguments you enter.

**Comment by jellz** [comment]
> I mean, with tab complete you can type in /fl and hit tab, I don't feel that its worth it to add a lobby only command for one or 2 more keys pressed.

> Yeah, I agree with Lukas. I don’t see the need in implementing it when it’s just two arguments you enter.

You can't type `/fl` and hit TAB. You need to use the full command `/friend(s) list`, the quickest way would be typing `/fr`, hitting TAB then SPACE, then typing `list` and pressing ENTER.

An `/fl` alias would be very convenient.

**Comment by Sniped** [comment]
The only thing I really have with it is that the aliases would be inconsistent across the network, why only have an alias for a network command on the lobby? 

> You can't type `/fl` and hit TAB. You need to use the full command `/friend(s) list`, the quickest way would be typing `/fr`, hitting TAB then SPACE, then typing `list` and pressing ENTER.

I was referring to the time it takes me to type `/friends list` manually rather than trying to tab it.

**Comment by jellz** [comment]
> The only thing I really have with it is that the aliases would be inconsistent across the network, why only have an alias for a network command on the lobby?

IMO it's not really anything to worry about. Players will work out it is only usable in the lobby and not in player servers.

**Comment by Sniped** [comment]
> IMO it's not really anything to worry about. Players will work out it is only usable in the lobby and not in player servers.

Yeah, I agree on that, it shouldn't really be a big deal if the alias only works in lobbies.
